### PR TITLE
nodejs-studio

### PR DESCRIPTION
**Pull request by MhdTa**

### CODE REVIEW

**Comment by karam-mustafa** [comment]
Hi muhammad, I suggest to put your linked in profile, instead of gmail email

File: package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "name": "nodejs-studio",
+    "version": "1.0.1",
+    "description": "",
```
Comment (karam-mustafa) [inline package.json]: Add description here please

File: package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "keywords": [
+        "github",
+        "repos",
+        "repositories",
+        "sort",
+        "stars"
+    ],
+    "author": "Yogesh Chavan <myogeshchavan97@gmail.com>",
```
Comment (karam-mustafa) [inline package.json]: update author details and license

**Review by karam-mustafa — CHANGES_REQUESTED** [review]

**Comment by karam-mustafa** [comment]
And you should specify the version for this PR and the description

**Comment by MhdTa** [comment]
done (:
Thank you, Mr.Karam

**Pull request merged**